src/logger.ts

```diff
@@ -1,8 +1,56 @@
 /**
- * A method that returns the default logger for this application.
- * Uses a function instead of a constant to allow mocking.
- * @returns The default logger for this application
+ * The default logger for this application.
+ * Can be set to any object that has `debug`, `info`, `warn`, and `error` methods,
+ * as long as it can be cast to a `Console`.
+ * @public
  */
-export function getLogger(): Console {
-	return console;
+const logger: Console = console;
+export default logger;
+
+/**
+ * A wrapper method for the `debug` endpoint of the default logger.
+ * In most situations, it wouldn't make sense to import a single logger endpoint,
+ * but separate exports are necessary for creating jest mocks for testing.
+ * @param message the message to print to `debug`
+ * @param optionalParams any additional messages or objects to print to `debug`
+ * @public
+ */
+export function debug(message?: never, ...optionalParams: Array<never>): void {
```

Comment:
Why `never`?

Reply:
Ah, good question. I wanted to make the wrapper function take the same parameters as `Console.debug`, which are `(message?: any, ...optionalParams: any[])`
![image](https://user-images.githubusercontent.com/28303477/194410398-4d21bc8f-c50c-4b4e-a4e6-2b5cc7521c13.png)

But TS didn't like me using the `any` type.
![image](https://user-images.githubusercontent.com/28303477/194410581-47100915-ae18-4609-be55-cecb8226cdf8.png)
![image](https://user-images.githubusercontent.com/28303477/194410642-44c35785-8112-4fc6-b2b4-389fe2602035.png)

One of the "Quick fix" suggestions was using the `unknown` type (which requires type-checking on every usage - not fun) or the `never` type (which fixed all the problems). The `never` type description says "this is useful when instantiating generic-type parameters that you don't need to know the type of", which is perfect for `Console.debug`.

Reply:
I'm not sure `unknown` is a bad idea here. The only thing that would need to do type-checking is the function's implementation, but since the value goes right to `console.debug`, which takes a parameter of type `any`, no type checking is necessary. If anything, `never` prevents _anything_ from being passed in without casting the function to something else first

Reply:
Ah, you're right, I didn't try to use the wrapper functions in a normal script until now. `unknown` works much better.